### PR TITLE
feat: module page initiate 

### PR DESCRIPTION
**Pull request by Rupantar-Borphukan**

### CODE REVIEW

File: src/pages/Module/Module.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import styles from "../Module/Module.module.css";
+import Module from "../../components/Module/Game";
```
Comment (codadept) [inline src/pages/Module/Module.jsx:4]: The default export name is Game and you are importing as Module?

File: src/pages/Module/Module.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import Footer from "../../components/Footer/Footer";
+
+const Module = () => {
+  return (
+      <>
+          <Module />
+          <Footer/>
```
Comment (codadept) [inline src/pages/Module/Module.jsx:11]: Footer should be common for all the pages so it should be present in App.jsx

File: src/components/Module/Game.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```
Comment (codadept) [inline src/components/Module/Game.jsx:1]: No need to import React

**Review by codadept — CHANGES_REQUESTED** [review]

File: src/pages/Module/Module.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import styles from "../Module/Module.module.css";
+import Module from "../../components/Module/Game";
+import Footer from "../../components/Footer/Footer";
```
Comment (codadept) [inline src/pages/Module/Module.jsx:5]: Chnage it to

```jsx
import { Footer } from "../../components";
```

**Review by codadept — CHANGES_REQUESTED** [review]

File: src/pages/Module/Module.jsx (hunk 2 above)
Comment (Rupantar-Borphukan) [reply src/pages/Module/Module.jsx:11]: ok bhaiya

File: src/components/Module/Game.jsx (hunk 3 above)
Comment (Rupantar-Borphukan) [reply src/components/Module/Game.jsx:1]: ok

File: src/pages/Module/Module.jsx (hunk 1 above)
Comment (Rupantar-Borphukan) [reply src/pages/Module/Module.jsx:4]: ok I will fix it

**Comment by codadept** [comment]
Closed due to inactivity

**Pull request closed without merging**